### ISSUE TITLE
Unable to estimate log_flux for 1-2 stars in a 10x10 tile. 

### ISSUE THREAD

**Issue by ismael-mendoza** [prob-ml/bliss#134]
Another potential bug that I wanted to document from the meeting. I find that currently bliss is able to find (up to) 2 stars in a 10x10 tile but not their log-fluxes. This was surprising to @Runjing-Liu120 given his previous studies. It will be interesting to find out if this is actually a bug.

@Runjing-Liu120 did you have a chance to test this in your old codebase? Thanks!

## Questions 

- Does work on 4x4 tiles? 

## How to reproduce issue? 

Here is some code that trains an encoder which has this issue. 

```
from hydra.experimental import initialize, compose
from bliss.datasets import simulated
from bliss import sleep

overrides = {
    'model':"basic_sleep_star_one_tile",
    'dataset':"single_tile",
    'training':"unittest", # avoid logging and plotting to speed up training.
}
overrides = [f"{key}={value}" for key, value in overrides.items()]
config_path = "../config"  # path to config folder relative to the notebook.

with initialize(config_path=config_path):
    cfg = compose("config", overrides=overrides)
    datamodule = simulated.SimulatedModule(cfg)
    sleep_net = sleep.SleepPhase(cfg)
    trainer = pl.Trainer(**cfg.training.trainer)
    trainer.fit(sleep_net, datamodule=datamodule)

trained_encoder = sleep_net.image_encoder
```

**Comment by jeff-regier** [comment]
@Runjing-Liu120 @ismael-mendoza Any indication whether this issue/bug applies to smaller tiles too? (I suspect it doesn't, but it would be helpful to have confirmation)

**Comment by Runjing-Liu120** [comment]
@jeff-regier I'll look into this, and let you know at our meeting tomorrow

**Comment by Runjing-Liu120** [comment]
Yeah I think I got it working, both on small tiles (2x2) and larger tiles (10x10). It just wasn't training enough before. 

I changed the batchsize from 64 to 6400 and the learning rate from 1e-5 to 1e-3. In other words, I updated the config files as such: 

```
cfg.optimizer.params.update({'lr': 1e-3})
cfg.dataset.params.update({'batch_size': 6400})
```

You can see my notebook here
[test_star_sleep.pdf](https://github.com/prob-ml/bliss/files/5631365/test_star_sleep.pdf)

**Comment by jeff-regier** [comment]
Thanks Bryan. Please remember to add a new unit test along with the batch size / learning rate changes

**Comment by Runjing-Liu120** [comment]
OK I'll add it -- pull request coming soon
